crates/egui_kittest/src/texture_to_bytes.rs

```diff
@@ -0,0 +1,83 @@
+use egui_wgpu::wgpu;
+use egui_wgpu::wgpu::{Device, Extent3d, Queue, Texture};
+use image::RgbaImage;
+use std::iter;
+use std::mem::size_of;
+use std::sync::mpsc::channel;
+
+pub(crate) fn texture_to_bytes(device: &Device, queue: &Queue, texture: &Texture) -> RgbaImage {
```

Comment:
A bit confusing name imho. Could be called `read_texture` or `image_from_texture`